src/wwwroot/styles/views/_settings.scss

```diff
@@ -24,6 +24,9 @@
           display: inline-block;
         }
       }
+      span.api-key--container p {
+        padding-right: 25px;
```

Comment:
Can we put this padding with 'em'? so when we need to reduce it in responsive mode, we can do it by changing only the font size of the body?